pkg/kubectl/cmd/create.go

```diff
@@ -167,6 +168,23 @@ See http://releases.k8s.io/HEAD/docs/user-guide/services-firewalls.md for more d
 				makePortsString(obj.Spec.Ports, true))
 			out.Write([]byte(msg))
 		}
+
+		_, ok := obj.Annotations[service.AnnotationLoadBalancerSourceRangesKey]
+		if ok && obj.Spec.SourceRange != "" {
+			msg := fmt.Sprintf(`You have specified both sourceRange field and annotation [service.beta.kubernetes.io/load-balancer-source-ranges].
```

Comment:
I do not think so. This happens upon `kubectl create`. I feel it is user-friendly.